go/vt/vtorc/inst/analysis_dao.go

```diff
@@ -468,6 +473,10 @@ func GetReplicationAnalysis(keyspace string, shard string, hints *ReplicationAna
 			a.Analysis = NotConnectedToPrimary
 			a.Description = "Not connected to the primary"
 			//
+		} else if topo.IsReplicaType(a.TabletType) && !a.IsPrimary && math.Round(a.HeartbeatInterval*2) != float64(a.ReplicaNetTimeout) {
```

Comment:
If the heartbeat interval is configured correctly then it should either be an integer or have `.5` as its fractional part because it is half of an integral value. Because we lose some precision, we're using math.Round to round it to the nearest integer after multiplying by 2.